clientapp/src/components/Card.jsx

```diff
@@ -11,6 +13,24 @@ startDate: string,
 endDate: string
 */
 
+const AddToCalendarDropdown = AddToCalendarHOC(Button, CalendarModal);
```

Comment:
I'm getting ReferenceError: Cannot access 'Button' before initialization. I think we need to move the Button and CalendarModal classes above this? I'm curious why you didn't run into this.

Reply:
What browser you use? I only test it in Chrome. However, when I rename button to CalendarButton, it is broken in both IE and Chrome.

Reply:
I tested in Edge. 

I think this should fix it.